usaspending_api/references/models.py

```diff
@@ -247,7 +250,7 @@ class LegalEntity(DataSourceTrackedModel):
     class Meta:
         managed = True
         db_table = 'legal_entity'
-        unique_together = (('recipient_name'),)
+        unique_together = (('recipient_unique_id'),)
```

Comment:
For creating a unique index on a single field, you can just specify `unique=True` when defining field, right? Any reason to put this down in `Meta`?